spec/factories/factory_bot.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'factory_bot'
+
+FactoryBot.define do
```

Comment:
A factory must be in line with her filename. Read [this](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rspec) please. Remember, how [factories](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#defining-factories) works.

Reply:
Also, read [this](https://semaphoreci.com/community/tutorials/working-effectively-with-data-factories-using-factorygirl).

Reply:
i did refactor this part of the code :)